main.py

```diff
@@ -62,9 +64,7 @@ def convert_email_to_id(email):
     return None
 
 def load_schedule_data():
```

Comment:
You don't really need load_schedule_data anymore in this case. Just use SCHEDULE_INFO directly.